app/views/components/phaseBanner.scala.html

```diff
@@ -0,0 +1,32 @@
+@*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+
+@this(govukPhaseBanner: GovukPhaseBanner)
+
+@(phase: String)(implicit request: Request[_], messages: Messages)
+
+@feedbackBanner = {
+ @messages("feedback.before")
```

Comment:
Pattern for I18N is to apply it twice
```
feedback = This is a new service - your {} will help us to improve it
feedback.link = feedback
```
with messages
```
link = <a href="???">messages("feedback.link")</a>
messages("feedback", link)
```

Reply:
Cool. Done!